src/Skeletor/Frameworks/Framework.php

```diff
@@ -105,14 +108,14 @@ public function setPaths(array $paths)
     public function configure()
     {
         $this->mountManager->copy(
-            'skeletor://'.$this->options['templatePath'].'/skeletor.md',
+            'skeletor://' . $this->options['templatePath'] . '/skeletor.md',
             'project://skeletor.md'
         );
     }
 
     public function install()
     {
         $command = $this->composerManager->prepareFrameworkCommand($this->getInstallSlug(), $this->getVersion());
-        $this->composerManager->runCommand($command);
+        $this->runManager->runCommand($command);
     }
```

Comment:
I like the new level of abstraction, its good to have a general `runManager` and not rely on the `composer`👍 
@titopixelfusion pointed this out before, but never got the time to fix it.